tycho-ethereum/src/entrypoint_tracer/balance_slot_detector.rs

```diff
@@ -1569,20 +1482,20 @@ mod tests {
         let usdc_bytes = alloy::hex::decode("A0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48").unwrap();
         let usdt_bytes = alloy::hex::decode("0xdAC17F958D2ee523a2206206994597C13D831ec7").unwrap();
 
-        let pool_manager_bytes =
-            alloy::hex::decode("000000000004444c5dc75cB358380D2e3dE08A90").unwrap();
+        let user_address_bytes =
+            alloy::hex::decode("f847a638E44186F3287ee9F8cAF73FF4d4B80784").unwrap();
 
         let weth = Address::from(weth_bytes);
         let usdc = Address::from(usdc_bytes);
         let usdt = Address::from(usdt_bytes);
 
-        let pool_manager = Address::from(pool_manager_bytes);
+        let user_address = Address::from(user_address_bytes);
```

Comment:
I'd prefer to keep the old test and parametrize to also test using the `user_address`.
The old test (pool_manager) tested the case where the owner had balance. In the new one, the user_address doesn't have balance, so they are 2 cases to be tested.